.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,10 +3,10 @@
   Before you make PR, please fill this form
 -->
 
-- **Title**: <!-- Write anime's title here -->
-- **Link**: <!-- Write MyAnimeList's link here -->
+- **Title**: Battle Programmer Shirase
```

Comment:
Don't touch this file. This template for pull request message. Just add the anime on README.md file

Reply:
okay sipp